test/server/integration/legend.spec.js

```diff
@@ -0,0 +1,72 @@
+describe('Legend', function() {
```

Comment:
I *think* this could use a couple more tests, though I'm definitely willing to discuss it:

1. There's nothing that checks that Doc Holliday correctly reduces your starting ghost rock. This one feels pretty important as it's a fairly unique modifier.
2. I think there should be a use of the Legend's ability.

Reply:
1. There is such test:
```
                it('should decrease wealth', function() {
                    expect(this.player1.player.ghostrock).toBe(18);
                });
```

2. Ability is really not that important, it is just a card with ability. I think what is more important is that if it has effect, it correctly affects cards in play. And there is a test for that.